### PR TITLE
Docs: Update user quick-start readme and fix scripts

### PR DESCRIPTION
**Pull request by momuno**
### Motivation and Context

  1. Why is this change required?  To make onboarding smoother.
  2. What problem does it solve?  The quick-start local deployment steps were difficult to follow.
  3. What scenario does it contribute to? Anyone trying out Chat Copilot locally for the first time.
  4. If it fixes an open issue, please link to the issue here. NA.

### Description

Main Chat Copilot Readme is updated to present a quick-start beginner use case.  This is the local deployment of Chat Copilot, not an Azure deployment.  It removes the redirect for the scripts, and formats instructions to improve readability.

Bash and PowerShell scripts are also fixed to use environment variable `ASPNETCORE_ENVIRONMENT`, which is necessary for appsettings.json override (by appsettings.Development.json) when running the backend for OpenAI.

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,173 +1,190 @@
...
 
-Refer to [./deploy/README.md](./deploy/README.md) for Azure configuration and deployment.
+# Setup Instructions
+## Windows
+1. Open PowerShell as an administrator.
+2. Configure environment.
 
-# Manual Setup and Local Deployment
+    ```powershell
+    cd <path to semantic-kernal>\samples\apps\copilot-chat-app\scripts
```
Comment (glahaye) [inline README.md:28]:
```suggestion
    cd <path to semantic-kernel>\samples\apps\copilot-chat-app\scripts
```

**Comment by momuno** [comment]
OpenAI and Azure OpenAI with Windows tested. Works.
Linux to be tested.

**Comment by momuno** [comment]
testing on linux and macos

**Comment by momuno** [comment]
closed in favor of PR #94

**Pull request closed without merging**